### ISSUE TITLE
Add documentation on how to setup the dag drawer for different OSs

### ISSUE THREAD

**Issue by 1ucian0** [Qiskit/qiskit#245]
_For reference, this was originally posted by @jaygambetta in https://github.com/Qiskit/qiskit-terra/issues/2281#issuecomment-489417445_

> @ajavadia and @mtreinish  it has been lost where to find how to add this dependencies outside pip. It is in the doc for the function https://github.com/Qiskit/qiskit-terra/blob/master/qiskit/visualization/dag_visualization.py but I think we need to make this clearer in the documentation in the Qiskit repo. 
>
> I would split this into two issues -- 
> 1. In terra add better error messaging. If you call drag_drawer and you don't have graphviz give that this dependency needs to be installed on your system. 
> 2. in qiskit add a documentation on how to setup the dag drawer for different operating systems.

This is issue is about the second item.

**Comment by jaygambetta** [comment]
I agree this needs to be done but it should  be put in the source code of the dag_drawer.  This is in terra.

**Comment by jaygambetta** [comment]
can you add to the source and it should be added here now?

https://qiskit.org/documentation/api/qiskit.visualization.dag_drawer.html#qiskit.visualization.dag_drawer

**Comment by 1ucian0** [comment]
Let me rephrase to be sure I'm following: you think the `qiskit.visualization.dag_drawer` docstring should include details in how to install Graphviz in different OSs. Is that correct? In that case, should this issue be moved to terra?

**Comment by jaygambetta** [comment]
yes and yes.
